### ISSUE TITLE
Bug in semantic similarity example?

### ISSUE THREAD

**Issue by xpact** [tensorflow/hub#95]
Re: https://github.com/tensorflow/hub/blob/master/examples/colab/semantic_similarity_with_tf_hub_universal_encoder.ipynb

Line: sim_scores = -tf.acos(tf.reduce_sum(tf.multiply(sts_encode1, sts_encode2), axis=1))
Should be:
sim_scores = 1-tf.acos(tf.reduce_sum(tf.multiply(sts_encode1, sts_encode2), axis=1))

as per formula in section 5
sim(u, v) = 1 − arccos (u · v/(||u|| ||v||))/π
in https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/46808.pdf

**Comment by akhorlin** [comment]
We have a fix pending for this issue. Once it is submitted we will update
the thread.
